index.js

```diff
@@ -0,0 +1,237 @@
+        this.elements.keys = this.elements.keysContainer.querySelectorAll('.keyboard__key');
+     
+        document.querySelectorAll('.use-keyboard-input').forEach(element=>{
+            element.addEventListener('focus',()=>{
+                this.open(element.value, currentValue=>{
+                    element.value = currentValue;
+                });
+                
+            });
+        });
+    },
+
+    _createKeys() {
+        const fragment = document.createElement('div');
+        // const keyLayoutEng = [
+        //     '`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+        //     'Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\', 'DEL',
+        //     'CapsLock', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '\'', 'ENTER',
+        //     'shift', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', 'top', 'Shift',
+        //     'Ctrl', 'Win', 'Alt', 'space', 'Alt', 'left', 'bottom', 'right', 'Ctrl'
+        // ];
+        const createIconHtml = (icon_name)=>{
+            return `<i class='material-icons'>${icon_name}</i>`;
+        };
+        for(let key in keyLayout){
+            const keyElement = document.createElement('button');
+            const insertLineBreak = ['Backspace', 'Delete', 'Enter','ShiftRight'].indexOf(keyLayout[key].code) !== -1;
+
+            keyElement.setAttribute('type','button');
+            keyElement.className = 'keyboard__key';
+            window.addEventListener('keydown',(e)=>{
+                if(e.keyCode == keyLayout[key].keyCode && e.code ==keyLayout[key].code){
+                    keyElement.classList.toggle('keyboard__key--dark');
+                }
+               
+            });
+            window.addEventListener('keyup',(e)=>{
+                if(e.keyCode == keyLayout[key].keyCode && e.code ==keyLayout[key].code){
+                    keyElement.classList.toggle('keyboard__key--dark');
+                }
+               
+            });
+            switch(keyLayout[key].but){
+            case 'Backspace':
+                keyElement.classList.add('keyboard__key--extra__wide');
+                keyElement.textContent = keyLayout[key].but;
+                keyElement.addEventListener('click',()=>{
+                    this.properties.value = this.properties.value.substring(0,this.properties.value.length-1);
+                    this._triggerEvent('oninput');
+                });
+                break;
+
+            case 'CapsLock':
+                keyElement.classList.add('keyboard__key--extra__wide', 'keyboard__key--activatable');
+                keyElement.textContent = keyLayout[key].but;
+                keyElement.addEventListener('click',()=>{
+                    this._toggleCapsLock();
+                    keyElement.classList.toggle('keyboard__key--active',this.properties.capsLock);
+                    
+                });
+                window.addEventListener('keydown',(e)=>{

```

Comment:
As I understand correctly you create listener on keydown event above. What do you need this for?